confignode/src/main/java/org/apache/iotdb/confignode/manager/ClusterSchemaManager.java

```
@@ -373,6 +376,34 @@ public TGetTemplateResp getTemplate(String req) {
     return resp;
   }
 
+  /**
+   * mount template
+   *
+   * @param templateName
+   * @param path
+   * @return
+   */
+  public TSStatus setSchemaTemplate(String templateName, String path) {
+    SetSchemaTemplatePlan setSchemaTemplatePlan = new SetSchemaTemplatePlan(templateName, path);
+    return getConsensusManager().write(setSchemaTemplatePlan).getStatus();
+  }
+
+  /**
+   * show path set template xx
+   *
+   * @param templateName
+   * @return
+   */
+  public List<String> getPathsSetTemplate(String templateName) {
+    GetPathsSetTemplatePlan getPathsSetTemplatePlan = new GetPathsSetTemplatePlan(templateName);
+    PathInfoResp resp =
+        (PathInfoResp) getConsensusManager().read(getPathsSetTemplatePlan).getDataset();
+    if (resp.getStatus().getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
+      return resp.getPathList();
+    }
+    return null;
```

If not success, shall we throw an exception or return a failure TSStatus?